src/api/json/catalog.json

```diff
@@ -302,7 +302,10 @@
       "name": "sarif.json",
       "description": "Static Analysis Results Interchange Format (SARIF)",
       "fileMatch": [ "*.sarif,*.sarif.json" ],
-      "url": "http://json.schemastore.org/sarif"
+      "url": "http://json.schemastore.org/sarif",
+      "versions": {
+        "1.0.0": "http://json.schemastore.org/sarif-1.0.0"
```

Comment:
Mads, am I correct that this catalog update and my file add of sarif-1.0.0.json will result in that latter schema being published to the following url? (which we emit as $schema information in latest tooling)

http://json.schemastore.org/sarif-1.0.0